templates/master/01-master-kubelet/_base/files/kubelet.yaml

```diff
@@ -7,7 +7,7 @@ contents:
     apiVersion: kubelet.config.k8s.io/v1beta1
     authentication:
       x509:
-        clientCAFile: /etc/kubernetes/ca.crt
```

Comment:
There are a lot of other references to this path, a quick ripgrep shows e.g. a bind in
`install/0000_80_machine-config-operator_04_deployment.yaml`.

Hm, but I guess previously...we were mixing the kubelet ca with the root ca?